app/components/Chatbot/index.tsx

```diff
@@ -145,13 +145,20 @@ type FollowupsProps = {
 }
 const Followups = ({title, followups, onSelect, className}: FollowupsProps) => {
   const {randomQuestions} = useOnSiteQuestions()
-  const items =
-    (followups?.length || 0) >= 3
-      ? followups
-      : [
-          ...(followups || []),
-          ...randomQuestions().map(({title, pageid}) => ({text: title, pageid})),
-        ].slice(0, 3)
+  const [items, setItems] = useState(followups)
+
+  useEffect(() => {
+    const initialItems =
+      (followups?.length || 0) >= 3
+        ? followups
+        : [
+            ...(followups || []),
+            ...randomQuestions().map(({title, pageid}) => ({text: title, pageid})),
+          ].slice(0, 3)
+
+    setItems(initialItems)
+    // eslint-disable-next-line
```

Comment:
I would prefer if `randomQuestions` was a stable reference to a useCallback-ed function that wouldn't mess with useEffect deps,, but not a blocker if that's not a trivial fix (I haven't checked myself at the moment)

Reply:
I agree, but yeah it might be difficult because `randomQuestions` comes from a hook -> `useOnSiteQuestions`